.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+language: python
+python:
+  - "2.7"
+
+install:
+  - pip install -r requirements.txt
+
+script:
+  # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
+  - ansible-playbook --syntax-check playbooks/byo/config.yml
+  # TODO(rhcarvalho): update make ci to pick up these tests
+  - nosetests test/
```

Comment:
@sdodson FYI I've made a change here to run only the tests in `test/`, while `make ci` picks up other tests (same as what runs in the 'default' Jenkins job).

I plan to improve this later when I'm back from PTO, for now it is already better than before :-)